packages/server-reason-react-ppx/DomProps.ml

```diff
@@ -688,7 +688,7 @@ let formHTMLAttributes =
     Attribute { name = "rel"; jsxName = "rel"; reasonJsxName = "rel"; type_ = String };
     Attribute { name = "enctype"; jsxName = "encType"; reasonJsxName = "encType"; type_ = String };
     Attribute { name = "action"; jsxName = "action"; reasonJsxName = "action"; type_ = String };
-    Attribute { name = "method"; jsxName = "method"; reasonJsxName = "method_"; type_ = String };
+    Attribute { name = "method"; jsxName = "method"; reasonJsxName = "method"; type_ = String };
```

Comment:
Still need to know how this is going to work on the SRR. But `reason-react` has it as "method"

Reply:
It's working on sample, but breaking on cram test